test/libsolidity/syntaxTests/globalFunctions/erc7201_with_bytes.sol

```diff
@@ -0,0 +1,8 @@
+bytes constant b = "abcdef";
+contract C {
+    function f() public pure returns (uint256) {
+        return erc7201(b);
+    }
+}
+// ----
+// TypeError 6896: (114-115): The argument of builtin erc7201 must be either a constant string variable or a string literal.
```

Comment:
I think it might be useful to have a separate message when someone uses `bytes` instead of a `string`. These are similar enough that some users might be confused (it actually took me a moment too to realize what the problem is here :)).